.github/docker/Dockerfile.gorgone-testing-alma9

```diff
@@ -0,0 +1,15 @@
+FROM almalinux:9
+
+RUN bash -e <<EOF
+
+dnf install -y dnf-plugins-core zstd mariadb iproute
+dnf install -y epel-release 
+dnf config-manager --set-enabled crb
+dnf config-manager --add-repo https://packages.centreon.com/rpm-standard/23.10/el9/centreon-23.10.repo
+dnf -y clean all --enablerepo=*
```

Comment:
This line seem uselless as you install other depedancy on the next line, and already clean dnf on line 13

Reply:
i removed the line 13 and kept the line 9

Reply:
You can't do that as you install again package line 10